examples/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"github.com/mainak90/colorify"
+	"io"
+	"os"
+)
+
+func main() {
+	// Int the struct first, now this struct can be reused for all further interface invocations.
+	// In general as string attributes are seldom changed, you can initialize them directly at the struct level.
+	col := colorify.Colorify{Attr: colorify.Bold}
+	colEmpty := colorify.Colorify{Attr: colorify.Regular}
+	// The color schemes are columnar, which now means you can use one color for out print output
+	//or as many colors as you like. Colors can be set directly on the interface function level making it easier.
+	col.Println(colorify.Red, "[Error]", colorify.Yellow, "This is a new error")
```

Comment:
This seems really nice!

Reply:
Thanks, guessed that this is the best way to do this